### PR TITLE
N15: Add learning materials section w/ bevy snake tutorial

### PR DESCRIPTION
**Pull request by marcusbuffett**
I couldn't figure out the stuff in the contributing document about reserving sections, but hopefully this is close enough... lmk if there's any changes needed.

_Part of #320 ("Newsletter 15")_

### CODE REVIEW

File: content/posts/newsletter-015/index.md [hunk 1]
```diff
@@ -62,6 +62,18 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### [Creating a Snake Clone in Bevy]
+
+![Snake clone tutorial output](bevy_snake.gif)
+_Gameplay of snake clone_
+
+Bevy is a rapidly growing game engine written in rust. This tutorial walks
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:70]:
```suggestion
Bevy is a rapidly growing game engine written in Rust. This tutorial walks
```

File: content/posts/newsletter-015/index.md [hunk 2]
```diff
@@ -62,6 +62,18 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+through creating a snake clone, introducing bevy concepts on the way. The
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:71]:
```suggestion
through creating a snake clone, introducing Bevy concepts on the way. The
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, thank you for the contribution :)

The bit about reserving sections is just to say that you should leave a comment on [the coordination issue](https://github.com/rust-gamedev/rust-gamedev.github.io/issues/320) if you're picking up a section, to avoid other people working on the same bit.

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**